portal-test/src/com/liferay/portal/test/rule/FeatureFlagTestRule.java

```diff
@@ -78,7 +78,7 @@ private Map<String, String> _enableFeatureFlags(Description description) {
 
 			PropsUtil.addProperties(
 				UnicodePropertiesBuilder.setProperty(
-					featureFlagKey, "true"
+					featureFlagKey, String.valueOf(featureFlags.enable())
```

Comment:
Since now we are handling false values, you must add a test case in `FeatureFlagTestRuleTest`